### DISCUSSION TITLE
Question regarding audio optimization

### DISCUSSION THREAD

**Discussion by forkHub** [mozilla/hubs#4935] [General Questions & Discussion]
Hi, I have question regarding the spatial audio setting and how it can affect the performance.

Can we edit the setting so that we can disable the spatial audio volume effect and let the user is restricted by distance only.

The scenario would be like this:
- User can hear others with the same volume regardless of the distance.
- User is limited to hear only persons who are near to them (like 10 m). More than 10m will be completely unheard.
 
How this will effect the performance?

I was just thinking by removing the spatial audio volume effect will increase the performance, because the server will have less work with the audio. It is ok to hear others in the same volume. 

I used to have a meeting daily with regular meeting apps and want to try to move to hubs, but I need to somehow increase the performance. I have optimized the room using low polygon, the light is baked, the avatar is also only a square. I think I have optimized everything. The thing left is how to optimze the audio. Is there anything I can do? It is ok to have only a base feature like roaming and talking for now. 

Thanks

**Comment by sobralfreenat** [comment]
Did you try audio debug panel?
Or you can simply tweak rolloff 
Let me know if you need help further  with this very important UX adjustment.
